pkg/controller/controller.go

```diff
@@ -67,7 +67,7 @@ type Config struct {
 	tenantReceiverSelector *metav1.LabelSelector
 	// Receiver for each tenant user, in form of map[tenantID]map[type/name]Receiver
 	receivers map[string]map[string]internal.Receiver
-	// Config for each tenant user, in form of map[tenantID]map[type/name]Receiver
+	// Controller for each tenant user, in form of map[tenantID]map[type/name]Config
```

Comment:
`Controller for each tenant user` => `Config for each tenant user` ?

Reply:
Done